File: Cosmos-IOS/Cosmostation/Cosmostation/Model/BnbHistory.swift
//
//  BnbHistory.swift
//  Cosmostation
//
//  Created by yongjoo on 01/10/2019.
//  Copyright © 2019 wannabit. All rights reserved.
//

import Foundation

public class BnbHistory {
    var txHash: String = ""
    var blockHeight: Int64 = -1
    var txType: String = ""
    var timeStamp: String = ""
    var fromAddr: String = ""
    var toAddr: String = ""
    
    init() {}
    
    init(_ dictionary: [String: Any]) {
        self.txHash = dictionary["txHash"] as? String ?? ""
        self.blockHeight = dictionary["blockHeight"] as? Int64 ?? -1
        self.txType = dictionary["txType"] as? String ?? ""
        self.timeStamp = dictionary["timeStamp"] as? String ?? ""
        self.fromAddr = dictionary["fromAddr"] as? String ?? ""
        self.toAddr = dictionary["toAddr"] as? String ?? ""
    }
}

File: app/src/main/java/wannabit/io/cosmostaion/model/type/BnbHistory.java
package wannabit.io.cosmostaion.model.type;

import com.google.gson.annotations.SerializedName;

public class BnbHistory {

    @SerializedName("txHash")
    public String txHash;

    @SerializedName("blockHeight")
    public int blockHeight;

    @SerializedName("txType")
    public String txType;

    @SerializedName("timeStamp")
    public String timeStamp;

    @SerializedName("fromAddr")
    public String fromAddr;

    @SerializedName("toAddr")
    public String toAddr;
}
